packages/wasm/crate/tests/test_insufficient_notes.rs

```diff
@@ -0,0 +1,124 @@
+mod utils;
+use crate::utils::planner_setup::seed_params_in_db;
+use decaf377::Fq;
+use penumbra_asset::asset::{Id, Metadata};
+use penumbra_asset::{Value, STAKING_TOKEN_ASSET_ID};
+use penumbra_num::Amount;
+use penumbra_proto::core::asset::v1 as pb;
+use penumbra_wasm::database::interface::Database;
+use penumbra_wasm::database::mock::{get_mock_tables, MockDb};
+use penumbra_wasm::planner::insufficient_funds_err;
+use penumbra_wasm::storage::{byte_array_to_base64, Storage};
+use rand_core::OsRng;
+use wasm_bindgen_test::wasm_bindgen_test;
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+use penumbra_proto::DomainType;
+
+#[wasm_bindgen_test]
+// Database throws an error while retrieving
```

Comment:
issue: this test hits the same case as `test_insufficient_notes_not_in_storage`. Triggering a database error is actually not supported in our mock db right now. To not confuse others, I'd leave a comment like:

```
// TODO: add a test for database error when mock storage can support errors
```